src/grisp_tools_build.erl

```diff
@@ -330,8 +330,16 @@ repo_sanity_check(Repo, S0) ->
 run_git_plumbings([], S) ->
     {up_to_date, S};
 run_git_plumbings([{Cmd,Check}|Plumbings], S0) ->
-    {{ok, Output}, S1} = shell(S0, Cmd),
-    case Check(Output) of
-        true -> run_git_plumbings(Plumbings, S1);
-        false -> {invalid, S1}
+    case shell(S0, Cmd, [return_on_error]) of
+        {{ok, Output}, S1} ->
+            case Check(Output) of
+                true -> run_git_plumbings(Plumbings, S1);
+                false -> {invalid, S1}
+            end;
+        {{error, { _, "fatal: unable to access 'https://github.com/erlang/otp/': "
+                        "Could not resolve host: github.com\n"}}, S1} ->
+            S1 = event(S0, [{error, Cmd}]),
+            run_git_plumbings(Plumbings, S1);
+        _ ->
+            throw(rebar_abort)
```

Comment:
Will this `throw` actually show the error, or should we just wait for a `cause_clause` error here instead?

Reply:
Actually is better to fail on pattern matching so we get the output.
The only difference is that is not printed by rebar in red as before...

Reply:
I don't follow. Matching on `_` will throw away the error...

Reply:
Just crash. We don't know which errors could occur here yet